src/dialogs.ycp

```diff
@@ -77,20 +77,20 @@ term EditDialogContents()
 					    `VBox(
 						  // combo box label: enable FCoE (yes/no)
 						  `ComboBox( `id(`fcoe), _("&FCoE Enable"),
-							     [`item(`id("yes"), "yes" ),
-							      `item(`id("no"), "no", true ) ]
+							     [`item(`id("yes"), _("Yes") ),
```

Comment:
Oh, OK, I didn't know that is is missing in SP4... Yes, it does not make much sense adding it.